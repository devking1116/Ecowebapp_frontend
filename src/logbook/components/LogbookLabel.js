/* @flow */
import React from 'react';
import { Label } from 'semantic-ui-react';

type Props = {
};

/**
 * A component that displays the Label marking a logbook.
 */
export default function LogbookLabel({ ...otherProps }: Props): React.Element<*> {
  //TODO, update to used diffrent colors depending on likned logbook
  return (
    <Label color="green" size="tiny" basic {...otherProps} />
  );
}
