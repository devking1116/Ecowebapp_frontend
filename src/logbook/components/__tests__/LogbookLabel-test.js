import React from 'react';
import renderer from 'react-test-renderer';

import LogbookLabel from '../LogbookLabel';

test('it displays the given children in a semantic-ui-react Label.', () => {
  const component = renderer.create(
    <LogbookLabel title="Test Title">
      Test Logbook
    </LogbookLabel>
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
