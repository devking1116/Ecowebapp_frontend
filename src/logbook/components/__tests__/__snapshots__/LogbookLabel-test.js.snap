exports[`test it displays the given children in a semantic-ui-react Label. 1`] = `
<div
  className="ui green tiny basic label"
  onClick={[Function]}
  title="Test Title">
  Test Logbook
</div>
`;
